styles/ContactUs.module.css

```diff
@@ -0,0 +1,70 @@
+.title {
+  text-align: center;
+  color: var(--primary-blue);
+  padding: var(--padding-lg);
+}
+.subtitle {
+  color: var(--primary-blue);
+}
+.subtitleBasedIn {
+  color: var(--primary-blue);
+  padding-bottom: 17.93px;
+}
+.mail {
+  text-decoration: none;
+  color: var(--primary-blue);
+  font-size: 1.25rem;
+}
+.mailimg {
+  visibility: hidden;
+}
+.address {
+  padding-top: 41px;
+  padding-bottom: 70px;
+}
+.midia {
+  padding-top: 41px;
+}
+.maple {
+  background-image: url(/vector2.png);
+  background-size: "cover";
+  height: 823px;
+  width: 825px;
+  margin: -713px 482px 9px;
```

Comment:
See the other comment about negative margins

Reply:
done